modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/snapshot/incoming/IncomingSnapshotCopier.java

```diff
@@ -68,6 +71,9 @@ public class IncomingSnapshotCopier extends SnapshotCopier {
 
     private static final int MAX_TX_DATA_BATCH_SIZE = 1000;
 
+    /** Number of milliseconds that the follower is allowed to try to catch up the required catalog version. */
+    private static final int WAIT_FOR_METADATA_CATCHUP_MS = 3000;
```

Comment:
Why don't we create proper configuration? What's up with all these constants?

Reply:
This one seems to be something really internal, I'm not sure it should be exposed and be tunable. Concerning other constants, we might create a JIRA ticket to think about them.